test/e2e/tests/test_endpoint.py

```diff
@@ -52,6 +53,8 @@ def single_container_model(name_suffix):
         replacements=replacements,
     )
     assert model_resource is not None
+    if k8s.get_resource_arn(model_resource) is None:
+        print(model_resource["status"])
```

Comment:
here and throughout, please use `logging.debug` instead of raw `print` statements.